services/storage/tests/unit/test_handlers_simcore_s3.py

```diff
@@ -564,6 +565,76 @@ async def test_create_and_delete_folders_from_project_burst(
     )
 
 
+@pytest.fixture
+async def user_files_ids(
```

Comment:
you might want to rename this to something like ```uploaded_file_ids``` as there are already fixtures creating SimcoreFileID that are not uploaded.